samples/ImplicitFlow/AuthorizationServer/.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "0.2.0",
```

Comment:
VS Code has a setting, in which it tries to detect the spacing preferences that are already in the file.